app/src/main/java/com/example/android/weatherapp/adapter/viewholder/ThreeHourlyWeatherViewHolder.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.android.weatherapp.adapter.viewholder
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.android.weatherapp.R
+import com.example.android.weatherapp.models.ThreeHourlyWeatherModel
+import kotlinx.android.synthetic.main.hourly_weather_layout.view.*
+import kotlinx.android.synthetic.main.three_hourly_weather_layout.view.*
+
+
+class ThreeHourlyWeatherViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    fun bindingViewToData(item: List<ThreeHourlyWeatherModel>) {
+        setFirstHourlyValues(item[0])
+        setSecondHourlyValues(item[1])
+        setThirdHourlyValues(item[2])
+        setFourthHourlyValues(item[3])
+        setFifthHourlyValues(item[4])
+    }
+
+    private fun setFirstHourlyValues(item: ThreeHourlyWeatherModel) {
```

Comment:
Create a generic method which takes data and view and updates it accordingly because there's too much repetition of same code here.

Reply:
Done. It was visible in next commit